### ISSUE TITLE
Disabling or Removing secondary windows

### ISSUE THREAD

**Issue by kaydwithers** [hpneo/gmaps#481]
Is it possible to disable or remove the secondary markers that are highlighted in yellow in the attached image? I only want my "HTML Content" markers clickable and have no other clickable landmarks or locations.

![untitled-1](https://cloud.githubusercontent.com/assets/1678609/13908267/4d427634-ef53-11e5-9abd-5796db446af6.jpg)

**Comment by hpneo** [comment]
I think you can override the map styles and hide the Point of Interests and/or transit stations. Check this demo: https://codepicnic.com/consoles/styled-maps

**Comment by kaydwithers** [comment]
Thanks @hpneo , the demo link doesn't work for me due to pricing. Do you have another demo?

I think this might work? I'm not sure where to put it in the API though.

`var myStyles =[
    {
        featureType: "poi",
        elementType: "labels",
        stylers: [
              { visibility: "off" }
        ]
    }
];
​
var myOptions = {
    zoom: 10,
    center: homeLatlng,
    mapTypeId: google.maps.MapTypeId.ROADMAP,
    styles: myStyles 
};`

**Comment by hpneo** [comment]
@kaydwithers it works now? anyway, the code is the following:

```javascript
var map = new GMaps({
el: "#map",
lat: 41.895465,
lng: 12.482324,
zoom: 15, 
zoomControl : true,
zoomControlOpt: {
  style : "SMALL",
  position: "TOP_LEFT"
},
panControl : true,
streetViewControl : false,
mapTypeControl: false,
overviewMapControl: false
});

var styles = [
  {
    stylers: [
      { hue: "#00ffe6" },
      { saturation: -20 }
    ]
  }, {
      featureType: "road",
      elementType: "geometry",
      stylers: [
          { lightness: 100 },
          { visibility: "simplified" }
    ]
  }, {
      featureType: "road",
      elementType: "labels",
      stylers: [
          { visibility: "off" }
    ]
  }
];

map.addStyle({
  styledMapName:"Styled Map",
  styles: styles,
  mapTypeId: "map_style"  
});

map.setStyle("map_style");
```

**Comment by kaydwithers** [comment]
Thanks @hpneo, I tried that but it turns off the POI's completely. I only want to turn off the option to make the POI clickable (Disable the pop-up). I think this has actually been an issue since 2012 that hasn't been addressed by Google yet. I found this post
https://code.google.com/p/gmaps-api-issues/issues/detail?id=3866

It looks like Google is looking into it, they have suggested using an experimental API to close POI's, so that 2 popups are not open at the same time. But that's not really what I want either.
